app/assets/javascripts/discourse/app/templates/wizard/step.gjs

```diff
@@ -0,0 +1,58 @@
+import RouteTemplate from "ember-route-template";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import Component from "@glimmer/component";
+import getUrl from "discourse-common/lib/get-url";
+import WizardCanvas from "discourse/static/wizard/components/wizard-canvas";
+import WizardStep from "discourse/static/wizard/components/wizard-step";
+
+export default RouteTemplate(
+  class extends Component {
+    <template>
+      {{#if this.showCanvas}}
+        <WizardCanvas />
+      {{/if}}
```

Comment:
We may even want to move this into WizardStep itself. It probably wan't possibly previously because the wrapping div was used for styling, but since I migrated it to glimmer component it should be no problem now